app/Http/Controllers/People/NotesController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Http\Controllers\People;
+
+use App\Note;
+use App\Contact;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\People\NotesRequest;
+
+class NotesController extends Controller
```

Comment:
`PeopleController.php` still has all the methods to manage a note. Is that an omission or on purpose?